core/math/math_funcs.h

```diff
@@ -178,6 +178,8 @@ class Math {
 		return abs(a - b) < CMP_EPSILON;
 	}
 
+	static _ALWAYS_INLINE_ bool is_whole_approx(real_t x) { return is_equal_approx(x, floor(x)); }
```

Comment:
Done. Also, made it round before casting to int, just in case it is better this way.